woocommerce/wc-admin
# Update how refunds are displayed in tables

Fixes #2002 

Adds styling to negative net totals and links refunds to parent order IDs.

**Dev**
* We could add the `is-negative` class conditionally to a span inside `formatCurrency()` but I'm not sure if this would be ideal.  Alternatively, we could also create a new function, something like `formatCurrencySpan()` that wraps the currency with classes given the value.

**Design**
* We decided not to add the tag `[Refund]` next to the order ID, right?
* Order statuses for refunds are always `Completed`- okay to keep this in the table?

/cc @josemarques @LevinMedia 

### Screenshots
<img width="1110" alt="Screen Shot 2019-04-23 at 7 36 29 PM" src="https://user-images.githubusercontent.com/10561050/56578006-1c74e700-65ff-11e9-847e-7bfaa749a0cb.png">


### Detailed test instructions:

1.  Drop the `wc_admin_version` option from your `wp_options` table to run the deltas on database tables.
2.  Create a refund.
3. Make sure the styling matches above for net totals.
4. Make sure order links go to the refund's parent order.
